unit-test/broker-protection.js

```diff
@@ -640,3 +646,29 @@ describe('utils', () => {
         })
     })
 })
+
+describe('create profile', () => {
+    fit('handles combined strings', () => {
+        const fac = (key) => {
+            return {
+                age: [{ innerText: 'John smith' }],
+                name: [{ innerText: 'John smith' }]
+            }[key]
+        }
+
+        const data = {
+            name: {
+                selector: '.name',
+                beforeText: ','
+            },
+            age: {
+                selector: '.name',
+                afterText: ','
+            }
+        }
```

Comment:
This format will now allow us to unit-test the extraction process from a high-level